build.sbt

```diff
@@ -41,3 +43,14 @@ scriptedLaunchOpts := {
     Seq("-Xmx1024M", "-Dplugin.version=" + version.value)
 }
 scriptedBufferLog := false
+
+// scalafix specific settings
+inThisBuild(
+  List(
+    semanticdbEnabled := true,
+    semanticdbVersion := scalafixSemanticdb.revision,
+    scalacOptions ++= Seq(
+      "-Ywarn-unused"
```

Comment:
Since scalafix uses scalac to determine if imports/values are used or not, this scalac compiler option is necessary.